bin/index.js

```diff
@@ -17,15 +17,19 @@ const cli = meow(`
         $ mdpdf README.md --footer footer.hbs --fHeight 22 --debug
 
     Options:
-        --style    A single css stylesheet you wish to apply to the PDF
-        --header   A HTML (.html) file to inject into the header of the PDF
-        --hHeight  The height of the header section in mm
-        --footer   A HTML (.html) file to inject into the footer of the PDF
-        --fHeight  The height of the footer section in mm
-		--noEmoji  Disables emoji conversions
```

Comment:
I didn't spot this, must have an editor configured incorrectly. Thanks for aligning them all.